W-11862525-first-draft-for-beta-site-build-CP (#114)
mulesoft/docs-apikit: modules/ROOT/pages/apikit-4-soap-fault-task.adoc

include::_attributes.adoc[]
endif::[]

> Use SoapUI to identify an operation you implement by making the API fail. Create a request in SoapUI using the `OrderTshirt` operation to place an order for a large t-shirt for John. As the API cannot handle this operation, a SOAP fault occurs.

Reviewer: I find this paragraph confusing. 

- Is "SoapUI" a term or should you say "Use the Soap UI...)?
- What is the goal? Is it to test what happens when an operation in the API fails or test an operation that only occurs when the API fails?
- Is OrderTshirt an example of an operation? If so, you need to introduce example - "For example...suppose you create a request in SoapUI using the...
- "As" > "Because" ("as" implies time, so it should not be used as a substitute for "because")

Author: I've made some changes. Please, let me know if I should make any other changes.
- SoupUI is software for making SOAP requests. In previous articles, the readers were requested to download the software.
- You try to know what happens with an implemented operation. To make this test, you make the API fail.
- Yes! `OrderTshirt` is part of the initial WSDL file. In previous articles, the readers were requested to download the software. And the reader is requested to follow the step. This step is not optional. The reader should not perform a task other than the one requested.
- OK! Thanks!

> To know what happens with an implemented operation, you make the API fail using SoapUI. Take the steps of the following example to create a request in SoapUI using the `OrderTshirt` operation to place an order for a large t-shirt for John. Because the API cannot handle this operation, a SOAP fault occurs.

Reviewer: ```suggestion
To know what happens with an implemented operation, make the API fail using SoapUI. Follow the steps in the following example to create a request in SoapUI using the `OrderTshirt` operation for a customer named `John`. Because the API cannot handle this operation, a SOAP fault occurs.
```

Reviewer: "Take the steps" is not the right wording. I know "Follow the steps in the following example..." has "follow" twice, but it is what you want the users to do.

Author: OK! Thanks!

> . In SoapUI, create a new SOAP project with the URL for your deployed Mule APIkit project in the initial WSDL file. +

Reviewer: ```suggestion
. In SoapUI, create a new SOAP project and put the URL for your deployed Mule APIkit project in the initial WSDL file. +
```

Author: OK! Thanks!

In this example, the URL is `+http://localhost:8081/TshirtService/TshirtServicePort?wsdl+`.
+
image::soapui-new-project.png["New SOAP project window displaying the initial WSDL file: http://localhost:8081/TshirtService/TshirtServicePort?wsdl"]
+

> . In SoapUI, request a large t-shirt for John by entering some values into the request envelope:

Reviewer: Maybe here, you ca say "request a large t-shirt for a customer named `John`...
